lib/solidus_braintree/engine.rb

```diff
@@ -22,6 +22,13 @@ class Engine < Rails::Engine
       end
     end
 
+    initializer 'add_solidus_braintree_response_to_log_entry_permitted_classes' do
+      Spree.config do |config|
+        config.log_entry_permitted_classes ||= []
```

Comment:
No need to initialize it as it's already an empty array by default: https://github.com/solidusio/solidus/blob/5d119ba3d1e05c508a6e991bdc5649174a6096c7/core/lib/spree/app_configuration.rb#L185

Reply:
@waiting-for-dev Okay, I'll remove that line and merge the PR :+1: